cmd/find.go

```diff
@@ -145,30 +115,95 @@ func doFindExec(ctx *cli.Context, fileContent contentMessage) {
 	console.Println(string(out.Bytes()))
 }
 
-// doFindWatch - enables listening on the input path, listens for all file/object
+// watchFind - enables listening on the input path, listens for all file/object
```

Comment:
Line 113 above, looks like unreachable code. If it is unreachable, we need to remove it.